[exporter/tinybird] Update readme (#41474)
open-telemetry/opentelemetry-collector-contrib: exporter/tinybirdexporter/README.md

[development]: https://github.com/open-telemetry/opentelemetry-collector/blob/main/docs/component-stability.md#development
<!-- end autogenerated section -->

## Description

The tinybird open telemetry expeorer exports telemetry data to [Tinybird](https://www.tinybird.co/) using the Events API. Tinybird is a real-time analytics platform that allows you to ingest, transform, and serve data at high speed.

> This exporter supports sending logs, metrics, and traces to Tinybird, where they can be analyzed and queried in real-time.

Reviewer: ```suggestion
This exporter sends logs, metrics, and traces to [Tinybird](https://www.tinybird.co/) via the [Events API](https://www.tinybird.co/docs/api-reference/events-api).

Tinybird is a real-time analytics platform for ingesting, transforming, and serving data with low latency and high throughput. Telemetry data sent to Tinybird can be queried and analyzed in real time using SQL.``

## Quick Start

> Looking for a quick start? Check our [Open Telemetry Template](https://github.com/tinybirdco/tinybird-otel-template)! where you will find a ready-to-go template to use the Tinybird Open Telemetry Exporter. 

Reviewer: ```suggestion
Looking for a quick start? Check out our [Open Telemetry Template](https://github.com/tinybirdco/tinybird-otel-template), which provides a ready-to-use example for configuring the Tinybird OpenTelemetry Exporter.
```

## Configuration

> 
> The following settings are required:
> 
> - `endpoint` (no default): The Tinybird API endpoint URL (e.g., `https://api.us-east.aws.tinybird.co`)
> - `token` (no default): Your Tinybird API token for authentication
> 

Reviewer: ```suggestion

### Required settings

- `endpoint` (no default): The Tinybird API endpoint URL (e.g., `https://api.us-east.aws.tinybird.co`)
- `token` (no default): Your Tinybird API token used for authentication

```

> The following settings can be optionally configured:

Reviewer: ```suggestion
### Optional settings
```

- `metrics`: Configuration for different metric types

>   - `gauge.datasource` (default: `gauge`): Name of the gauge metrics data source
>   - `sum.datasource` (default: `sum`): Name of the sum metrics data source  
>   - `histogram.datasource` (default: `histogram`): Name of the histogram metrics data source
>   - `exponential_histogram.datasource` (default: `exponential_histogram`): Name of the exponential histogram metrics data source
> - `traces.datasource` (default: `traces`): Name of the traces data source

Reviewer 2: You should use `::` instead of `.` for namespacing, `.` will not work
```suggestion
  - `gauge::datasource` (default: `gauge`): Name of the gauge metrics data source
  - `sum::datasource` (default: `sum`): Name of the sum metrics data source  
  - `histogram::datasource` (default: `histogram`): Name of the histogram metrics data source
  - `exponential_histogram::datasource` (default: `exponential_histogram`): Name of the exponential histogram metrics data source
- `traces::datasource` (default: `traces`): Name of the traces data source
```

        datasource: "metrics_histogram"
      exponential_histogram:
        datasource: "metrics_exponential_histogram"
    traces:
      datasource: "traces"
    logs:
      datasource: "logs"
    retry_on_failure:
      enabled: true
    sending_queue:
      enabled: true

>       queue_size: 104857600     # 100 MB total queue

Reviewer: ```suggestion
      queue_size: 104857600     # 100 MB total buffer size
```

maybe?
